src/defrag.c

```diff
@@ -690,6 +690,13 @@ int scanLaterStreamListpacks(robj *ob, unsigned long *cursor, monotime endtime)
         server.stat_active_defrag_scanned++;
         if (++iterations > 128) {
             if (getMonotonicUs() > endtime) {
+                /* Move to next node. */
+                if (!raxNext(&ri)) {
```

Comment:
maybe we should explain the reason here (without reading commit comment this code may look odd and someone might "fix" it).
or maybe the better place for this comment is where we do `>=`? i'm not sure.

Reply:
The other place is `/* Move to next node. */`, it doesn't matter where to put it, but if someone fixes it, both will be changed.
done with [`3923c14` (#13847)](https://github.com/redis/redis/pull/13847/commits/3923c14479ec1a3e7e1f9aa2038a4d84a03bab4a)